packages/dashboard/src/utils/constants.ts

```diff
@@ -0,0 +1,12 @@
+export const INTERACTIVE_REQUESTS = [
+  "eth_signTransaction",
+  "eth_sendTransaction",
+  "eth_decrypt",
+  "eth_signTypedData",
+  "eth_signTypedData_v1",
+  "eth_signTypedData_v3",
+  "eth_signTypedData_v4",
+  "personal_sign"
+];
+
+export const UNSUPPORTED_REQUESTS = ["eth_sign"];
```

Comment:
what's wrong with this one?

Reply:
eth_sign silently fails with metamask, also when using it without the dashboard-provider. personal_sign works though. I can definitely document this better though. What do you think the best place is for that?

Reply:
probably just give a more specific error message for this case, rather than a single bucket for everything unsupported.

unless we're being vague to conform with existing JSON-RPC vagueness